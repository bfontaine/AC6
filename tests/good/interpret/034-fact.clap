def fact (x) = if x = 0 then 1 else x * fact (x - 1)
val z = fact 5

