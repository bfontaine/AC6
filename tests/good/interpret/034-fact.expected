:- fact = <code>
:- z = 120
